src/translator/scorers.cpp

```diff
@@ -72,6 +73,18 @@ std::vector<Ptr<Scorer>> createScorers(Ptr<Options> options) {
       LOG(warn, "No model settings found in model file");
     }
 
+    // check if there are left-to-right and right-to-left models together
+    if(models.size() > 1 && modelOptions->has("right-left")) {
+      if(i == 0) {
+        isPrevRightLeft = modelOptions->get<bool>("right-left");
+      } else {
+        // abort as soon as there are two consecutive models with opposite directions
+        ABORT_IF(isPrevRightLeft != modelOptions->get<bool>("right-left"),
+                 "A left-to-right and right-to-left model cannot be decoded together");
```

Comment:
Better: "Left-to-right and right-to-left models cannot be used together in ensembles"

Reply:
Changed.